### PR TITLE
Refactor and comment downloaded and compiled toolchains

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Move the common parts of br-ext/configs/toolchain-aarch32-sdk and br-ext/configs/toolchain-aarch64-sdk into
br-ext/configs/toolchain-common-sdk.
br-ext/configs/toolchain-aarch32-sdk was previously using BR2_TOOLCHAIN_EXTERNAL_GCC_11=y and is now by using br-ext/configs/toolchain-common-sdk updated to use the correct BR2_TOOLCHAIN_EXTERNAL_GCC_14=y.

Add comments where to update if compiled or downloaded toolchain is updated.

### CODE REVIEW

File: toolchain.mk [hunk 1]
```diff
@@ -57,11 +57,14 @@ endef
 
 ifeq ($(UNAME_M),x86_64)
 ifeq ($(ARCH),arm)
+# Please keep in sync with br-ext/configs/toolchain-aarch32
+# and below for x86_64 host
```
Comment (jforissier) [inline toolchain.mk:61]: You mean for aarch64 host?

File: br-ext/configs/sdk-common [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# Please keep GCC version and headers in sync with
+# br-ext/configs/toolchain-external-sdk
```
Comment (jforissier) [inline br-ext/configs/sdk-common:2]: You mean `toolchain-common-sdk`?

**Review by jforissier — COMMENTED** [review]
Two comments below, other than that looks good.

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: toolchain.mk (hunk 1 above)
Comment (jenswi-linaro) [reply toolchain.mk:61]: You're right, I'll update.

**Comment by jenswi-linaro** [comment]
Comments addressed and tag applied.

**Pull request merged**